src/pages/studyView/StudyViewPage.tsx

```diff
@@ -386,7 +386,9 @@ export default class StudyViewPage extends React.Component<
                 {this.store.queriedSampleIdentifiers.isComplete &&
                     this.store.invalidSampleIds.isComplete &&
                     this.store.unknownQueriedIds.isComplete &&
-                    this.store.displayedStudies.isComplete && (
+                    this.store.displayedStudies.isComplete &&
+                    this.store.queriedPhysicalStudies.isComplete &&
```

Comment:
i'm curious to know why this wouldn't be caught by unknownQueriedIds (above).  Maybe the study list service returns the ID even if user doesn't have access to it?